File: JAVA/chapter_2/26-jun/ProfitLoss.java
import java.util.*;

public class ProfitLoss{
	public static void main(String x[]){


		Scanner sc = new Scanner (System.in);
		int sell,cost;

		System.out.println("Enter selling price" );
		sell = sc.nextInt();

		System.out.println("Enter cost price " );
		cost = sc.nextInt();

		if(sell > cost)  { 
		System.out.println("Profit is ");  }

		else {
			System.out.println("Loss ");
 			
              }
}

}

File: C/1-july/profitloss.cpp
// program to input cost price and selling price of a product and check profit or loss.
#include<stdio.h>

int main(){
	int sp,cp;
	printf("Enter Selling Price ");
	scanf("%d",&sp);
	printf("Enter Cost Price ");
	scanf("%d",&cp);
	(sp>cp) ? printf("Profit ") : printf(" Loss ");	
}
